packages/browser-wallet/src/popup/shared/Toast/Toast.tsx

```diff
@@ -0,0 +1,48 @@
+import { toastsAtom } from '@popup/state';
+import clsx from 'clsx';
+import { useAtom } from 'jotai';
+import React, { useEffect, useState } from 'react';
+
+// The timeouts have to be aligned with the animations in the corresponding CSS.
+const toastTimeoutMs = 5400;
+const fadeoutTimeoutMs = 400;
```

Comment:
I have clarified it a bit. Basically it required manually tweaking until it looked right. The fadeout has to be a little less than the CSS animation to avoid weird UI behavior.